releases/2022.3/2022.3.md

```diff
@@ -0,0 +1,150 @@
+- Upgrade Kytos-ng core dependencies to leverage upstream fixes, enhancements and be able to leverage more asyncio functionalities
+- All Kytos-ng NApps should have at least 85% of unit test coverage
+
+In order to achieve the objectives of this version, the following enhancements were refined and delivered:
+
+### 1. `mef_eline` should be able to create EVCs with constrained paths
+
+`mef_eline` now supports EVC creation with path constraints that can be applied to a dynamic primary path or secondary dynamic failover paths. The following link metadata contrainsts can be used if they're configured: `bandwidth`, `utilization`, `priority`, `reliability`, `delay` and `ownership`. In addition other path attributes such as total accumulated cost, desired links and undesired links are also supported. The parameters are documented on [mef_eline OpenAPI specification](https://kytos-ng.github.io/api/mef_eline.html#operation/create_circuit).
+
+In addition, `mef_eline` provides service level prioritization to allow EVCs to be handled based on their descending `service_level` value during convergence and consistency checks.
+
+### 2. Specify how INT will be integrated and supported via NoviFlow experimenter actions
+
+A new `telemetry` has been specified on blueprint [EP031](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP031.rst), which will be responsible for enabling INT on EVCs. NoviFlow switches will be supported via experimenter actions that been implemented by the `noviflow` NApp, and the functionality will be optionally enabled when an EVC is being created. The `telemetry` NApp is expected to be available in the `2023.1` release.
+
+### 3. `maintenance` stabilization v1 API and integrate with MongoDB 
+
+**TODO** fill out what will be feasible to be delivered, ask David to update this paragraph
+
+### 4. Support NoviFlow switches on end-to-end tests
+
+NoviFlow switches are now supported on [kytos-end-to-end-tests](https://github.com/amlight/kytos-end-to-end-tests/) as a target on pre-defined [testing topologies](https://github.com/amlight/kytos-end-to-end-tests/blob/feature/noviswitch/tests/helpers.py). Currently, NoviFlow switches are managed via SSH using `paramiko` and this integration was designed to be seamless with mininet, NoviFlow switches are handled as a mininet `Switch` so you can reuse or write new end-to-end tests.
+
+### 5. end-to-end tests should have fewer false positives and support retries
+
+end-to-end tests were enhanced to avoid performing assertions when network convergence can still be taking place, minimizing false positives asserts that could happen. Also, `pytest-rerunfailures` was introduced to help out renunning tests, that way providing additional evidence facilitating to identify and isolate underlying issues.
+
+### 6. `flow_stats` stabilization v1 API
+
+`flow_stats` has had its API reviewed and augmented consolidating its v1 API. `flow_stats` has simplified its code and responsibilities.
```

Comment:
@gretelliz feel free to add or complement any other relevant information here if needed. Thanks